### ISSUE TITLE
Metacello installation broken

### ISSUE THREAD

**Issue by bachitoph** [pavel-krivanek/VoxelSpace#1]
Evaluating 

Metacello new
  baseline: 'VoxelSpace';
  repository: 'github://pavel-krivanek/VoxelSpace/src';
  load.

gives error

Could not resolve: BaselineOfVoxelSpace.

Windows 7 - Fresh downloaded Pharo 6.1 Windows

**Comment by pavel-krivanek** [comment]
The problem is not in the VoxelSpace code but in Pharo 6.1 that has currently broken Metacello version without Tonel format support. It should be fixed this week. You can try Pharo 7 instead of it.

**Comment by bachitoph** [comment]
Sorry for this. But loading the Pharo 7 image sml and sources. And starting it with the Pharo6.exe in Windows gets me on two different computers (win7 and win10) the fault message

Pharo cannot write to the ..changes file.
Asking me having no write permission. One time I got a walk back with cannot write to stdout.
 I know, this is not your turn, but maybe you have any Idea. I tried it also starting as administrator.

Ignoring the message and starting
Metacello new
  baseline: 'VoxelSpace';
  repository: 'github://pavel-krivanek/VoxelSpace/src';
  load.
Gives Walkback LGit_GIT_ERROR: error authenticating: failed connecting agent

any ideas
